tests/integration/integration_test.py

```diff
@@ -1248,41 +1187,59 @@ def resolve_envs(_conf):
             _conf[k] = os.getenv(v[1:])
 
 
+test_modes = ['consumer', 'producer', 'performance', 'admin',
+              'avro', 'avro-https', 'avro-basic-auth',
+              'throttle']
+
+
+def print_usage(exitcode, reason=None):
+    """ Print usage and exit with exitcode """
+    if reason is not None:
+        print('Error: %s' % reason)
+    print('Usage: %s [options] <testconf>' % sys.argv[0])
+    print('Options:')
+    print(' %s - limit to matching tests' % ', '.join(['--' + x for x in test_modes]))
+
+    sys.exit(exitcode)
+
+
 if __name__ == '__main__':
     """Run test suites"""
 
     if with_pympler:
         tr = tracker.SummaryTracker()
-        print('Running with pympler memory tracker')
+        print("Running with pympler memory tracker")
 
     modes = list()
 
     # Parse options
     while len(sys.argv) > 1 and sys.argv[1].startswith('--'):
         opt = sys.argv.pop(1)[2:]
 
-        if opt == 'conf':
-            testconf = sys.argv.pop(1)
-            continue
+        if opt == "help":
+            print_usage(0, None)
 
-        if opt not in all_modes:
+        if opt not in test_modes:
             print_usage(1, 'unknown option --' + opt)
         modes.append(opt)
 
-    with open(testconf) as f:
+    if len(sys.argv) == 2:
+        testconf_file = sys.argv.pop(1)
+
+    with open(testconf_file) as f:
         testconf = json.load(f)
         resolve_envs(testconf)
 
-    bootstrap_servers = testconf.get('bootstrap.servers', None)
-    topic = testconf.get('topic', None)
-    schema_registry_url = testconf.get('schema.registry.url', None)
+    bootstrap_servers = testconf.get('bootstrap.servers')
+    topic = testconf.get('topic')
+    schema_registry_url = testconf.get('schema.registry.url')
 
     if len(modes) == 0:
-        modes = default_modes
+        modes = test_modes
 
     if bootstrap_servers is None or topic is None:
         print_usage(1, "Properties bootstrap.servers and topic must be set. "
-                       "Use tests/testconf-example.json as a template when creating a new conf file.")
+                       "Use {} as a template when creating a new conf file.".format(testconf_file))
```

Comment:
is testconf_file the example/template file?

Reply:
With the use of docker I think it's best to just inform the user to ensure their testconf includes bootstrap.servers

Reply:
i.e it's no longer a template or an example it is a configuration file which you can edit as needed.